package.json

```diff
@@ -29,80 +29,80 @@
-    "@iconify/ember": "^2.1.0",
-    "broccoli-asset-rev": "^3.0.0",
-    "concurrently": "^8.2.2",
-    "ember-auto-import": "^2.6.3",
-    "ember-cli": "~5.4.0",
-    "ember-cli-app-version": "^6.0.1",
-    "ember-cli-babel": "^8.2.0",
-    "ember-cli-clean-css": "^3.0.0",
-    "ember-cli-clipboard": "^1.0.0",
-    "ember-cli-dependency-checker": "^3.3.2",
-    "ember-cli-fastboot": "^4.1.1",
-    "ember-cli-htmlbars": "^6.3.0",
-    "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "^4.0.2",
-    "ember-click-outside": "^5.0.1",
-    "ember-composable-helpers": "^5.0.0",
-    "ember-data": "~5.3.0",
-    "ember-fetch": "^8.1.2",
-    "ember-in-viewport": "^4.1.0",
-    "ember-keyboard": "^8.2.0",
-    "ember-load-initializers": "^2.1.2",
-    "ember-modifier": "^4.1.0",
-    "ember-page-title": "^8.0.0",
-    "ember-qunit": "^8.0.1",
-    "ember-ref-bucket": "^5.0.1",
-    "ember-resolver": "^11.0.1",
-    "ember-set-helper": "^2.0.1",
-    "ember-source": "~5.4.0",
-    "ember-template-lint": "^5.11.2",
-    "ember-toastr": "^3.0.1",
-    "ember-truth-helpers": "^3.1.1",
-    "ember-welcome-page": "^7.0.2",
-    "eslint": "^8.52.0",
-    "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-ember": "^11.11.1",
-    "eslint-plugin-n": "^16.2.0",
-    "eslint-plugin-prettier": "^5.0.1",
-    "eslint-plugin-qunit": "^8.0.1",
-    "loader.js": "^4.7.0",
-    "local-ssl-proxy": "^2.0.5",
-    "pre-commit": "^1.2.2",
-    "prettier": "^3.0.3",
-    "qunit": "^2.20.0",
-    "qunit-dom": "^2.0.0",
-    "stylelint": "^15.11.0",
-    "stylelint-config-standard": "^34.0.0",
-    "stylelint-prettier": "^4.0.2",
-    "toastr": "^2.1.4",
-    "tracked-built-ins": "^3.3.0",
-    "webpack": "^5.89.0"
+    "@babel/core": "7.24.7",
+    "@babel/eslint-parser": "7.24.7",
+    "@babel/plugin-proposal-decorators": "7.24.7",
+    "@ember/jquery": "2.0.0",
+    "@ember/optional-features": "2.1.0",
+    "@ember/render-modifiers": "2.0.5",
+    "@ember/string": "3.1.1",
+    "@ember/test-helpers": "3.3.0",
+    "@embroider/compat": "3.0.0",
+    "@embroider/core": "3.0.0",
+    "@embroider/webpack": "3.0.0",
+    "@fortawesome/ember-fontawesome": "0.4.1",
+    "@fortawesome/free-solid-svg-icons": "6.2.0",
+    "@glimmer/component": "1.1.2",
+    "@glimmer/tracking": "1.1.2",
+    "@iconify/ember": "2.1.0",
+    "broccoli-asset-rev": "3.0.0",
+    "concurrently": "8.2.2",
+    "ember-auto-import": "2.7.3",
+    "ember-cli": "~5.9.0",
+    "ember-cli-app-version": "6.0.1",
+    "ember-cli-babel": "8.2.0",
+    "ember-cli-clean-css": "3.0.0",
+    "ember-cli-clipboard": "1.0.0",
+    "ember-cli-dependency-checker": "3.3.2",
+    "ember-cli-fastboot": "4.1.1",
+    "ember-cli-htmlbars": "6.3.0",
+    "ember-cli-inject-live-reload": "2.1.0",
+    "ember-cli-sri": "2.1.1",
+    "ember-cli-terser": "4.0.2",
+    "ember-click-outside": "5.0.1",
+    "ember-composable-helpers": "5.0.0",
+    "ember-data": "~5.3.3",

```

Comment:
do we really  need this "~"

Reply:
No, We don't need ^ as well as ~, the reson behind this whenever a new package can be upgraded it can be done as it is without additional checks that ^ and ~ provides, which makes it easier to upgrade the packages.